forth_src/defer.fs

```diff
@@ -12,3 +12,5 @@ dup latest @ - ( nt size )
 latest @ swap over +  ( srca dsta )
 dup latest !
 r> move then ;
+hide >body
```

Comment:
Oops, this is used in `testcore`.  It could be redefined in compat and `>body` factored away here.

Reply:
I'll review this once everything is resolved :-)